test/fixture_test.js

```diff
@@ -578,6 +577,30 @@ test('fixture.store can use id of different type (#742)', function () {
 		});
 });
 
+test('fixture("METHOD /path", store) should use the right method', function () {
+	/*
+		Examples:
+			fixture("GET /path", store) => fixture("GET /path", store.getData)
+			fixture("POST /path", store) => fixture("GET /path", store.createData)
+	*/
+
+	// NOTE: this is a copy-paste of the test case
+	//       "fixture.store can use id of different type (#742)"
+	var store = fixture.store(100, function (i) {
+		return {
+			id: i,
+			name: 'Object ' + i
+		};
+	});
+	fixture('GET /models', store); // <- CHANGE
```

Comment:
The behavior is tested. I don't think testing the warning is necessary. In the next major version we can make the warning a hard error in which case it would be worth testing (and much easier to test).